.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
-  - "lts/*"
-
+- lts/*
 after_script:
-  - npm run coveralls
+- npm run coveralls
+env:
+  global:
+    secure: S/C5661Lm9LD/NVXg4h6lA8Fy2y3suAprXVqz9eycCM0Vspb5f7XClvS+8wdizC2Q63rqhGJe6gkeNiJ+KUsAPLrk8gIKvg7alNJS9ebyUZaHfxy0HYjBj2eDtVY8ISJXoaBP7dFWSjwjw6dX8HVKIIjwidvF3EEjTF0jue4q54xTKERcRXZBwIV/PnupgLjSFgjvkUSgAD+SK0tLfqoExI9rHFOqugFh1l3u6p+WPV7GbQWktOFyrYoC0SZzLPUBLz5KBin/AwnHFFHg7nabObSEOSvcqH73Gp6vu4dodjZONGsI90K5KRwHhZ4SSt28f2L0XgEarFlhN1/COtEr2ceJN1HgpbPy/R8D11vIGz0fvIKJcPuONTv5XWHYStkyGpKxkbR5Dbfs9xz4PZlBZGD8o1OQGkWQMZoLGDh5f3F/L1Ntz9YyKkST7IIY35hy578Dzmm3ykRgqfxwucj1b2Cxhr/mKvyib8AUCcCEYDwYtPw8++x2AXdQR/WJ3+M9y6GzJmGVfmwdJbcc2uAaSbXrLnQhVGCFNTHnxSnjb+oxwvObixsDzROlcOfKrHs2SUk9cuQHYqnAHuAWeLIoLzn1P3QXWvL16fVUsUn3VVAdrkBioR1Btb4hGaZF0CuXUFh0cMBrqXImmY/y26Xd2Qvo+qyybjlbFOogthu5ko=
```

Comment:
What env is this? is this your personal AV API key in travis ci?

Reply:
This is a new one I generated for Travis CI - it's encoded so shouldn't be accessible except for the platform